apps/notifications/tests/services/test_notification.py

```diff
@@ -0,0 +1,115 @@
+import json
+import os
+import uuid
+from unittest import mock
+
+from django.core.exceptions import ObjectDoesNotExist
+from django.test import TestCase
+from django.utils import timezone
+
+from apps.notifications.models.payload import Payload
+from apps.notifications.services.notification import NotificationService
+from apps.notifications.services.payload import PayloadService
+from apps.subscribers.models.subscriber import Subscriber
+from apps.subscribers.services.subscriber import SubscriberService
+
```

Comment:
nit: lets convert these to relative imports. Relative imports within the same app. Absolute if you are accessing methods from a different app